AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -152,6 +152,9 @@ filewhopper.com##.cl_js_inited
 pornpics.com###cookie-contract
 labbe.de##.acris-cookie-consent
 labbe.de##.ec-gtm-cookie-directive
+doordash.com#$#.cassie-cookie-module { display: none!important; }
+doordash.com#$#html[style*="overflow:"] { overflow: auto!important; }
+doordash.com#$#body { overflow: auto!important; }
```

Comment:
```
doordash.com#$#.cassie-cookie-module { display: none !important; }
doordash.com#$#html[style*="overflow:"] { overflow: auto !important; }
doordash.com#$#body[style*="overflow:"] { overflow: auto !important; }
```
Please take a look at the difference.

Reply:
Btw, please also check this `||cscript-cdn-use.cassiecloud.com/loader.js` instead of the rules above.